task_03.py

```diff
@@ -0,0 +1,84 @@
+# /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Task 03"""
+
+import decimal
+
+NAME = raw_input('What is your name? ')
+P = int(raw_input('What is the amount of your principal? '))
+T = int(raw_input('For how many years is this loan being borrowed? '))
+PREQ = raw_input('Are you pre-qualified for this loan? ')
+
+if 0 <= P <= 199999 and 1 <= T <= 15:
+    if PREQ == 'Yes' or 'y':
+        R = 3.63
+
+    else:
+        if PREQ == 'No' or 'N':
+            R = 4.65
+
+if 0 <= P <= 199999 and 16 <= T <= 20:
```

Comment:
I think there's more nuance here -- you could have nested these into a a three-layer condition:

```
PREQ = PREQ.lower()[0] == 'y'

if 0 <= P <= 199999:
    if 1 <= T <= 15:
        if PREQ:
        else:
    elif 16 <= T <= 20:
```